app/workers/metrics/record_daily_usage_worker.rb

```diff
@@ -40,7 +51,7 @@ def record_active_days_of_group(ids_by_day, user_ids, group)
       distinct_user_values = user_ids_by_day.flatten.group_by(&:itself).transform_values(&:count).values
       distinct_counts = distinct_user_values.group_by(&:itself).transform_values(&:count)
       distinct_counts.keys.each do |key|
-        DataDogStatsClient.gauge("users.#{group}_active_#{key}_days_past_7_days", distinct_counts[key], tags: { resource: "users" })
+        DataDogStatsClient.count("users.#{group}_active_#{key}_days_past_7_days", distinct_counts[key], tags: { resource: "users" })
```

Comment:
Still think you are going to have an easier time here if you put the day count in a tag and not the metric name....

Reply:
Okay so we can name these all `users.active_past_7_days` and have both the `group` and `key` elements here as tags with a payload along the lines of: `{group: group, count_segment: key, resource: "users"}`

And then in Datadog we select `users.active_past_7_days` and then the matching tags for what we want to chart?

Reply:
So the key/metric name is `users.past_active_days` then the tag is `day_count` which is set to a value of 1, 2, 3, 4, 5, 6, or 7 and then when you make a graph. You select `users.past_active_days` and avg_by `day_count`

For example, here we have the metric `trace.sidekiq.job.hits` then each one of those has the tag `resource_name` which is the name of the job so we select the metric `trace.sidekiq.job.hits`  and then avg_by the tag, `resource_name` and you have all your counts on one graph.

<img width="1070" alt="Screen Shot 2020-01-17 at 6 22 43 PM" src="https://user-images.githubusercontent.com/1813380/72652906-ad8ea980-3956-11ea-82dd-48cb3abab74d.png">

OR if you are in the metric explorer you select that same metric and you break out different graphs for each different `resource_name`

<img width="1430" alt="Screen Shot 2020-01-17 at 6 25 40 PM" src="https://user-images.githubusercontent.com/1813380/72652949-e038a200-3956-11ea-9c07-98c43dd8534f.png">

Let me know if that makes sense.